test/parallel/test-stream-writable-acceptance.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+
+const { stream, Writable } = require('stream');
+const { once } = require('events');
+
+async function test() {
+    const o = new http.OutgoingMessage();
+    assert.strictEqual(o instanceof http.OutgoingMessage, true);
+    assert.strictEqual(o instanceof Writable, true);
+
+    let external;
+
+    const server = http.createServer(async (req, nodeStreamResponse) => {
+        const isOutgoingMessage = nodeStreamResponse instanceof http.OutgoingMessage;
+        //const isStream = stream.isInstance(nodeStreamResponse);
+        const isWritable = nodeStreamResponse instanceof Writable;
+
+        assert.strictEqual(nodeStreamResponse.writable, true);
+        assert.strictEqual(isOutgoingMessage, true);
+        assert.strictEqual(isWritable, true);
+
+
+        const b = typeof nodeStreamResponse?._writableState
```

Comment:
This and other lines won't pass the linter. Make sure you run `make test` locally to make sure all other tests pass in addition to the linting.